search/elasticsearch/impl/src/test/org/sakaiproject/search/elasticsearch/ElasticSearchTest.java

```diff
@@ -134,8 +130,6 @@ public void createTestResources() {
             resources.put(name, resource1);
             events.add(newEvent);
             when(newEvent.getResource()).thenReturn(resource1.getName());
-            when(newEvent.getContext()).thenReturn(siteId);
```

Comment:
You can configure the old behaviour but I think it's cleaner to have only the required mocks